File: src/ToolbarStyles.ml
open Glamor

let root = css [
  background "#263238";
]

let button = css [
  background "#263238";
  color "#aaa";
  padding "1em";

  Selector("& .mdi", [
    marginRight ".25em";
  ]);

  Selector("&:hover", [
    background "#364248";
  ]);
]

File: lib/js/src/ToolbarStyles.js
'use strict';

var Colors       = require("./styles/Colors.js");
var Glamor       = require("bs-glamor/lib/js/src/glamor.js");
var ButtonStyles = require("./styles/ButtonStyles.js");

var root = Glamor.css(/* :: */[
      Glamor.background(Colors.panel),
      /* [] */0
    ]);

var button = ButtonStyles.button;

exports.root   = root;
exports.button = button;
/* root Not a pure module */
